lib/internal/process/execution.js

```diff
@@ -153,12 +149,12 @@ function createOnGlobalUncaughtException() {
         const report = internalBinding('report');
         if (report != null && report.shouldReportOnUncaughtException()) {
           report.writeReport(
-            typeof er?.message === 'string' ?
+            (er && typeof er.message === 'string') ?
```

Comment:
Why not keep the optional binding operator here? That seems contradictory with the change below: we should either treat `er` as a possibly falsy value (and use `er &&` and `er ||`), or as a possibly nullish value (and use `er?.message` and `er ??`).

Reply:
The reason I didn't keep the optional chaining operator there for an optional binding is to check if the value of the `er` parameter is not nullish and then proceed to check for it's `message` property type and compare it, when the `er` parameter is nullish, it'll get the type of the `er` parameter itself instead of the `message` property, which does a falsy comparison, which might be a little better to first check if it's truthy then compare the type?

Reply:
Why not just `er?.message ?? 'Exception'`? Can `message` be something other than a string?

Reply:
> Why not just `er?.message ?? 'Exception'`? Can `message` be something other than a string?

Not sure, but something else other than an `Error` instance is probably passed in or something.

Reply:
> when the er parameter is nullish, it'll get the type of the er parameter itself instead of the message property

That’s not correct, if `er` is nullish, `er?.message` is `undefined`.
`er?.message` will be of type `string` if and only if ˋer` is not nullish and has a property message of type ˋstring`. I really think we should keep it as is.

Reply:
@aduh95 
> That’s not correct, if `er` is nullish, `er?.message` is undefined.

That's what I meant to say really as it'll return `undefined` if `er` is nullish. 😅

> I really think we should keep it as is.

Reverting it now.